pkg/ccl/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -1568,3 +1594,148 @@ func TestUserPrivileges(t *testing.T) {
 		jobutils.WaitForJobToPause(t, dbA, jobAID)
 	})
 }
+
+func TestDLQ(t *testing.T) {
```

Comment:
any reason you put this in logical_replication_job_test instead of in dead_letter_queue_test ?

Reply:
I thought that `logical_replication_job_test`  might be a better place for e2e tests but it doesn't test any ldr job functionalities. Moved the test to `dead_letter_queue_test` instead